webservice/routes/api.php

```diff
@@ -3,11 +3,13 @@
 Route::group([
     'middleware' => ['cors', 'api'],
 ], function () {
-    Route::post('/login', 'LoginController@login');
+    Route::post('/auth/issue', 'AuthController@issueToken');
```

Comment:
This was a suggestion from @hernandev

Follows the original text:

Keep in mind that that's is not synonymous login, the route could be something like /auth/ issue instead of login, since the login is what happens when the api reads the request token, and not when it's issued.